doc/level1.rst

```diff
@@ -161,42 +161,40 @@ NOT be included in the cleartext of the Autocrypt Setup Message, or
 otherwise transmitted over e-mail.
 
 An Autocrypt Level 1 client MUST generate a setup code as UTF-8 string
-of 36 numeric characters, divided into nine blocks of
-four, separated by dashes. The dashes are part of the secret
-code and there are no spaces. This format holds about 119 bits of entropy. It is designed to be
-unambiguous, pronounceable, script-independent (chinese, cyrillic etc.),
-easily input on a mobile device and split into blocks that are easily
-kept in short term memory. For instance::
-
-    1203-1923-2307-
+of 36 numeric characters, divided into nine blocks of four, separated
+by dashes. The dashes are part of the secret code and there are no
+spaces. This format holds about 119 bits of entropy. It is designed to
+be unambiguous, pronounceable, script-independent (chinese, cyrillic
+etc.), easily input on a mobile device and split into blocks that are
+easily kept in short term memory. For instance::
+
+    9503-1923-2307-
     1980-7833-0983-
     1998-7562-1111
 
 An Autocrypt Setup Message payload that uses this structure for its
-setup code SHOULD include the following ``Passphrase-Format`` header
-in the outer OpenPGP armor::
+setup code SHOULD include a ``Passphrase-Format`` header with value
+``numeric9x4`` in the ASCII-armored data. This allows providing a
+specialized input form during decryption, with greatly improved
+usability.
 
-    Passphrase-Format: numeric9x4
+As a further measure to improve usability, it is RECOMMENDED to reveal
+the first two digits of the first block in a ``Passphrase-Begin``
+header, sacrificing about 7 bits of entropy. Those digits can be
+pre-filled during decryption, which reassures the user that they have
+the correct code before typing the full 36 digits. It also helps
+mitigate a possible type of phishing attack that asks the user to
+input their setup code.
 
-This OpenPGP header MUST NOT be present with this value if the Setup
-Code does not match the format described above.
+The headers might look like this::
 
-Note: if the autocrypt implentation makes the passphrase easier
-to read by inserting spaces around the dashes, then the implemenation
-should warn the user that spaces are not part of the passphrase, because
-not all OpenPGP implementations make it easy for the autocrypt implementation
-to filter the passphrase.  (GnuPG, for instance, uses pinentry to retreive the passphrase, which
-means the autocrypt implementation cannot check the passphrase's format.)
```

Comment:
I also though over this idea and did not like the general recommendation to the user to be warned not to enter spaces:
- if the GUI does not allow the to enter spaces at all, this looks weird
- other MUA may have the possibility to filter the password before decrypting.
- only if both does not work on the destination device, a hint is usefull - but then this should be done on the destination device. Should we recommend this hint to the user?